text/0000-style-team.md

```diff
@@ -0,0 +1,71 @@
+- Making determinations about styling for new Rust constructs
+- Evolving the existing Rust style
+- Defining mechanisms to evolve the Rust style while taking backwards compatibility into account, such as via Rust editions or similar mechanisms
+
+## Team structure and membership
+
+The Rust style team will be a subteam of both the rustfmt team and the Rust language team. (For the purposes of structures that currently require a team to have only one parent team, such as the `rust-lang/team` repository, the Rust style team will have the rustfmt team as its parent team.)
+
+The initial members of the style team shall be:
+- Lead: Caleb Cartwright (@calebcartwright)
+- Jane Losare-Lusby (@yaahc)
+- Josh Triplett (@joshtriplett)
+- Michael Goulet (@compiler-errors)
+
+The Rust style team shall have at least 3 members and at most 8. If the team has fewer than 3 members it shall seek new members as its primary focus.
+
+Members of the style team are nominated by existing members. All existing members of the team must affirmatively agree to the addition of a member, with zero objections; if there is any objection to a nomination, the new member will not be not added. In addition, the team lead or another team member will check with the moderation team regarding any person nominated for membership, to provide an avenue for awareness of concerns or red flags.
+
+The style team will have regular synchronous meetings when it has work to do. (The style team may also choose to handle individual agenda items asynchronously.) The style team shall not meet when it does not have work to do, but it shall remain in existence.
+
+The style team will use labels such as `T-style` and `I-style-nominated` on rust-lang repositories, to identify and handle issues requiring style decisions.
+
+The output of the Rust style team shall be modifications to the Rust style guide, and other guidance to the rustfmt team. The style team shall also make determinations regarding changes to the existing style, typically in the form of proposed changes to rustfmt options; such changes shall be applied in new Rust editions or via similar mechanisms, to avoid generating churn and CI failures in existing Rust code.
+
+Note that the Rust style guide will generally match the latest version of the Rust style; the style team does not plan to maintain multiple branches of the style guide for different editions, in part because formatting for new constructs will apply to any edition supporting those constructs.
+
+This RFC proposes to move the Rust style guide to the rust-lang/rust repository, rather than its current location in the RFCs repository. Style work may additionally take place in the `fmt-rfcs` repository (which this RFC proposes to revive). Larger style proposals may wish to start there rather than as PRs to the style guide. The style team may choose to change this process.
```

Comment:
Perhaps a deep link would be helpful here
```suggestion
This RFC proposes to move the Rust style guide to the rust-lang/rust repository, rather than its current location in the RFCs repository. Style work may additionally take place in the `fmt-rfcs` [repository](https://github.com/rust-dev-tools/fmt-rfcs) (which this RFC proposes to revive). Larger style proposals may wish to start there rather than as PRs to the style guide. The style team may choose to change this process.
```